.snippets/code/get-started/start-building/batch/batch-jsonl-01.py

```diff
@@ -0,0 +1,109 @@
+from openai import OpenAI
+from getpass import getpass
+import json
+import time
+
+# Get API key from user input
+api_key = getpass("Enter your kluster.ai API key: ")
```

Comment:
We should probably use the "INSERT_API_KEY" syntax. We only really do the above in the colab notebooks

Reply:
I gave this some thought and I prefer `getpass` as it is more secure and colab also uses it. That is why I modified it

Reply:
Does this prompt you in the terminal or something?